src/core/settings.py

```diff
@@ -241,8 +243,9 @@ def _load_api_raml(path):
 
 INGESTION_LOG_NAME = 'ingestion-%s.log' % PROJECT_NAME
 
-LOG_FILE = join('/var/log/', LOG_NAME) # ll: /var/log/lax.log
-INGESTION_LOG_FILE = join('/var/log/', INGESTION_LOG_NAME) # ll: /var/log/lax.log
+LOG_DIR = '/var/log/' if ENV != DEV else './'
```

Comment:
handled here: https://github.com/elifesciences/lax-formula/commit/8e8b5d637ef5b4e071fe20e994b18885b8a689ac

it's a neat little trick, we might want to consider it for some of our other projects